openpype/hosts/houdini/plugins/publish/extract_mantra_ifd.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+import pyblish.api
+
+from openpype.pipeline import publish
+
+import hou
+
+
+class ExtractMantraIFD(publish.Extractor):
+
+    order = pyblish.api.ExtractorOrder
+    label = "Extract Mantra ifd"
+    hosts = ["houdini"]
+    families = ["mantraifd"]
+    targets = ["local", "remote"]
+
+    def process(self, instance):
+        if instance.data.get("farm"):
+            self.log.debug("Should be processed on farm, skipping.")
+            return
+
+        ropnode = hou.node(instance.data.get("instance_node"))
+        output = ropnode.evalParm("soho_diskfile")
+        staging_dir = os.path.dirname(output)
+        instance.data["stagingDir"] = staging_dir
+        file_name = os.path.basename(output)
+
+        files = instance.data["frames"]
+        missing = []
```

Comment:
This could be:
```py
missing = [
	frame
	for frame in instance.data["frames"]
	if not os.path.exists(os.path.normpath(os.path.join(staging_dir, frame)))
]
```